lib/blacklight/hierarchy/engine.rb

```diff
@@ -4,8 +4,6 @@
 module Blacklight
   module Hierarchy
     class Engine < Rails::Engine
-      config.closed_icon = '⊞'
```

Comment:
This would be a breaking change requiring a major version bump.

Reply:
What functionality is breaking?

Reply:
I think that's probably fine -- especially given the very limited number of consumers (none of which customize this icon?).

Reply:
Major version increments are free though